LayoutTests/requestidlecallback/requestidlecallback-deadline-cap-by-rendering-update.html

```diff
@@ -25,7 +25,7 @@
     setTimeout(() => {
         shouldBeTrue('didRunIdleCallback');
         finishJSTest();
-    }, 200);
+    }, 1000);
```

Comment:
Would it be reasonable to do something like this, instead of a hard-coded timeout?

```shouldBecome("didRunIdleCallback", "true", finishJSTest);```

Reply:
Good idea. Will do.